pymc_marketing/model_config.py

```diff
@@ -392,9 +462,11 @@ def create_distribution(
     -------
     TensorVariable
         A PyMC random variable.
-
     """
-    dim_handler = create_dim_handler(kwargs.get("dims"))
+    if centered_hierarchy is False:
```

Comment:
Need to check that dist=Normal and because that is assumed

Reply:
Done!